.github/workflows/ci.yaml

```diff
@@ -163,11 +175,6 @@ jobs:
       - name: Install requirements
         run: |
           ./scripts/ci/install-requirements.sh
-      - name: Run Redis Service Container
-        timeout-minutes: 2
-        run: |
-          docker run --rm --detach -p 127.0.0.1:6379:6379/tcp --name redis redis:latest
-          until [ "$(docker inspect -f {{.State.Running}} redis)" == "true" ]; do sleep 0.1; done
```

Comment:
We only need one redis container, either started here or in `services` above. The `services` approach is a bit nicer because we know that redis is up and responding (thanks to the health check that GHA waits for before continuing), whereas this just makes ssure the container (not redis in the container) is running.

This was also the cause of the conflicts where `redis` was already in use. Getting rid of this allows us to use the simple `redis` name for the service container instead of `redis-server`.